docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -1312,6 +1312,96 @@ NOTE: Advanced, features (e.g. `ServiceBiniding` mappings) are not yet supported
+| mysql      | link:https://operatorhub.io/operator/percona-xtradb-cluster-operator[Percona XtraDB Cluster] | pxc.percona.com/v1-9-0                    | PerconaXtraDBCluster |
+| mongo      | link:https://operatorhub.io/operator/percona-server-mongodb-operator[Percona Mongo]          | psmdb.percona.com/v1-9-0                  | PerconaServerMongoDB |
+|====
+
+
+==== Automatic datasource binding
+
+For traditional databases, automatic binding is initated whenever a datasource is configured. For example:
+
+[source,properties]
+----
+quarkus.datasource.db-kind=postgresql
+----
+
+When the above configuration is provided and `quarkus-datasource`, `quarkus-jdbc-postgresql`, `quarkus-kubernetes` and `quarkus-kubernetes-service-binding` extensions are added to the application, then a `ServiceBinding` for `postgresql` will be generated.
+The generated `ServiceBinding` will bind to kubernetes resource with `apiVersion` and `kind` that matches the `postgresql` operator (see the table above) and `name` equal to `postgresql`. Since, the configured datasource is unamed, the `db-kind` is used.
+
+[source,yaml]
+----
+  services:
+  - apiVersion: postgres-operator.crunchydata.com/v1beta1
+    kind: PostgresCluster
+    name: postgresql
+----
+
+For named datasources, the name of the datasource is used instead:
+
+[source,properties]
+----
+quarkus.datasource.fruits-db.db-kind=postgresql
+----
+
+The the `service` in the generated `ServiceBinding` will look like:
+
+[source,yaml]
+----
+  services:
+  - apiVersion: postgres-operator.crunchydata.com/v1beta1
+    kind: PostgresCluster
+    name: fruits-db
+----
+
+Similarly, if mysql is used:
+
+[source,properties]
+----
+quarkus.datasource.fruits-db.db-kind=mysql
+----
+
+The generated `service` will be:
+
+[source,yaml]
+----
+  services:
+  - apiVersion: pxc.percona.com/v1-9-0
+    kind: PerconaXtraDBCluster
+    name: fruits-db
+----
+
+==== Customizing automatic bindings
+
+There are cases where customization upon the generated `ServiceBinding` may be needed. For example, the target resource name does not match the datasource name, or a non default operator needs to be used.
+For such cases, it's possible to use the `quarkus.kubernetes-service-binding.servies.<service id>.` properties and specify as service id the name of the service.
+
+The example below shows how to customize the `apiVerison` of the `PostgresCluster` resource:
+
+[source,properties]
+----
+quarkus.datasource.db-kind=postgresql
+quarkus.kubernetes-service-binding.services.postgresql.api-version: postgres-operator.crunchydata.com/v1beta2
+----
+
+Or in case that we have a named datasource:
+
+[source,properties]
+----
+quarkus.datasource.fruits-db.db-kind=postgresql
+quarkus.kubernetes-service-binding.services.fruits-db.api-version: postgres-operator.crunchydata.com/v1beta2
```

Comment:
Same here

Reply:
I am not sure I follow

Reply:
This line doesn't look like a valid `properties` line - it looks like a mix between properties and YAML